testhelpers/docker.go

```diff
@@ -49,12 +49,13 @@ func DockerRun(t *testing.T, image string, ops ...DockerCmdOp) string {
 	return Run(t, exec.Command("docker", append([]string{"run", "--rm"}, args...)...))
 }
 
-func DockerRunAndCopy(t *testing.T, containerName, copyDir, image, path string, ops ...DockerCmdOp) {
+func DockerRunAndCopy(t *testing.T, containerName, copyDir, image, path string, ops ...DockerCmdOp) string {
```

Comment:
Why did we make this change?
I don't see that we're using the output of this function anywhere.

Reply:
IIRC we were using it for debugging...